sphinxlint/__main__.py

```diff
@@ -157,11 +184,14 @@ def main(argv=None):
         for path in chain.from_iterable(walk(path, args.ignore) for path in args.paths)
     ]
 
+    sort_order = args.sort_by.split(",") if args.sort_by else []
     if len(todo) < 8:
-        count = print_results(starmap(check_file, todo))
+        errors = sort_errors(starmap(check_file, todo), sort_order)
+        count = print_errors(errors, sort_order)
```

Comment:
```suggestion
        errors = starmap(check_file, todo)
        count = print_errors(sort_errors(errors, sort_order), sort_order)
```

I prefer it that way, instead of mixing "doing the things and sort the results" I prefer having one line about "doing the thing" and one line about "presenting the results".

It applies for the `else:` too.

Reply:
will do